src/cluster.c

```diff
@@ -778,6 +778,33 @@ int isValidAuxString(char *s, unsigned int length) {
     return 1;
 }
 
+int verifyPortNumber(client *c, long long *port, long long *cport) {
```

Comment:
Is there a reason this function is in cluster.c/h instead of static in cluster_legacy.c? It seems to be used only in one place.

Reply:
The function resetClusterStats() , isNodeAvailable() , clusterNodeIsMyself() are declared in cluster.h, and defined in cluster.c, I just follow the rule.